config/clouddriver.yml

```diff
@@ -51,6 +51,13 @@ google:
       jsonPath: ${providers.google.primaryCredentials.jsonPath}
       consul:
         enabled: ${providers.google.primaryCredentials.consul.enabled:false}
+      # Specify the project id of each GCP project whose images you want indexed and
```

Comment:
Do you think it's worth adding this link? https://cloud.google.com/compute/docs/images/sharing-images-across-projects#granting_a_user_access_to_images